src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (a) => { // getSum(5)
+    count++;
+
+    return (b) => { // ( getSum(5) )(6)
+      if (count <= 3) { // first three works everytime
+        return a + b;
+      } else if (count % 2) { // odd (НЕпарні) works correct
```

Comment:
you can combine these two conditions in one if-statement (count <= 3 and count % 2)